config/500-watcher.yaml

```diff
@@ -44,5 +44,22 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-
-# don't remove this line
```

Comment:
you are going to break the release.yaml generation if you remove them,

Reply:
I am getting lint error for that line
```
./config/500-watcher.yaml
  67:1      warning  comment not indented like content  (comments-indentation)
```

Reply:
fixed it.